docs/reference/analysis/analyzers/stop-analyzer.asciidoc

```diff
@@ -110,14 +116,36 @@ The `stop` analyzer accepts the following parameters:
 [horizontal]
 `stopwords`::
 
-    A pre-defined stop words list like `_english_` or an array  containing a
+    A pre-defined stop words list like `_english_` or an array containing a
     list of stop words.  Defaults to `_english_`.
 
 `stopwords_path`::
 
     The path to a file containing stop words. This path is relative to the
     Elasticsearch `config` directory.
 
+The default English stopwords used in Elasticsearch are:
+
+    a, an, and, are, as, at, be, but, by, for, if, in, into, is, it,
```

Comment:
Is this still the case?

I know Lucene has a file of EN stop words here:
https://github.com/apache/lucene-solr/blob/master/lucene/analysis/common/src/resources/org/apache/lucene/analysis/snowball/english_stop.txt

Might be worth checking with an engineer in @elastic/es-search.

Reply:
It also may make more sense to move this section under the `stopwords` param. Or link to this section from there.

Reply:
The list looks good, the file you linked is for the snowball filters and analyzers but we don't use them in the `stop` filter. The default list can be found here: https://github.com/apache/lucene-solr/blob/master/lucene/analysis/common/src/java/org/apache/lucene/analysis/en/EnglishAnalyzer.java#L46